backend/src/service.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * service.ts
+ *
+ * scipnet - Multi-tenant writing wiki software
+ * Copyright (C) 2019 not_a_seagull, Ammon Smith
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// registry of services, as well as functions for executing those services
+import * as fs from "fs";
+import * as jayson from "jayson";
```

Comment:
This `*` import seems unnecessary. Why not
```suggestion
import { Server as JaysonServer } from "jayson";
```
or something